tidy3d/components/data/monitor_data.py

```diff
@@ -589,8 +589,13 @@ def poynting(self) -> ScalarFieldDataArray:
         tan_fields = self._colocated_tangential_fields
         dim1, dim2 = self._tangential_dims
 
-        e1_h2 = tan_fields["E" + dim1] * tan_fields["H" + dim2].conj()
-        e2_h1 = tan_fields["E" + dim2] * tan_fields["H" + dim1].conj()
+        e1 = tan_fields["E" + dim1]
+        e2 = tan_fields["E" + dim2]
+        h1 = tan_fields["H" + dim1]
+        h2 = tan_fields["H" + dim2]
+
+        e1_h2 = e1 * (np.real(h2) - 1j * np.imag(h2))
```

Comment:
should we overwrite the `DataArray.conj()` method to do this? or is it better to just leave it here for now?

Reply:
Yeah we probably should, good point.

Reply:
Actually I think it might be better to keep it like this for this PR, otherwise we run into the `.attrs` vs `.values` handling again.

Reply:
Nevermind, implemented it.